includes/components/class-admin.php

```diff
@@ -946,6 +949,261 @@ protected function get_extensions_tab() {
 		return $current_tab;
 	}
 
+	/**
+	 * Gets system report.
+	 *
+	 * @return array
+	 */
+	protected function get_report() {
```

Comment:
Please cleanup this function to make it as optimal as possible, for example:

1. Avoid adding custom translatable strings (this will reduce our % of translation on WP.org, translations are available there with 90%+ translated so adding custom strings requires hiring translators again and possibly disables the available language until updated). 

For this, also make sure that if you add a string, it's as re-usable as possible, e.g. instead of "HivePress version", "WooCommerce version" use a placeholder "%s version" with the plugin name via sprintf.

Also, instead of "yes/no" strings you can use icons (check maybe WordPress has some styled way to use them, e.g. in WordPress/Dashboard section they show some icons) https://developer.wordpress.org/resource/dashicons/

You can also use a "no" icon instead of a custom error message if it's the only error message related to a feature, e.g. for cURL it basically means "not available" so icon is ok, a single custom error message is not needed when it's obvious

Reply:
Overall, please reduce and simplify the code, use as few custom strings as possible (you can search for similar strings in Loco to re-use when available), also maybe WooCommerce implemented this a long time ago, e.g. for post types you can use this https://developer.wordpress.org/reference/functions/wp_count_posts/ instead of a raw SQL query.